rtt_rosparam/README.md

```diff
@@ -63,6 +64,7 @@ policy (see below).
 ##### Operations for setting single properties
 
 * **setParam(ros_name, rtt_name)** Set the ROS param **ros_name** from the value in the RTT property **rtt_name**. Use leaders like `~` and `/` for private and absolute resolution.
+* `setParam_type(name, value)` Sets the value of the ROS param `name` to `value`. Use leaders like `~` and `/` for private and absolute resolution.
```

Comment:
```suggestion
* `setParam<Type>(name, value)` Sets the value of the ROS param `name` of type `<type>` to `value`. Use leaders like `~` and `/` for private and absolute resolution.
```

and it also needs to be moved to a separate section because it does not set a property.